preview/__test__/buffer_itr.test.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+'use strict';
+
+const bufferItr = require("../buffer_itr.js");
+
+const bufferedCollect = async itrGen => {
+  const all = [];
+  for await (const c of bufferItr(itrGen(), 20)) {
```

Comment:
Is there any rhyme or reason to "long" being defined as >20? Might be worth dropping a comment here to be clear.

Reply:
Just because it is long enough to trigger the grouping. I'll add a comment.